evennia/utils/containers.py

```diff
@@ -106,6 +106,7 @@ class GlobalScriptContainer(Container):
         callables from settings but a custom dict of tuples.
 
     """
+    __BASE_SCRIPT_TYPECLASS = class_from_module(settings.BASE_SCRIPT_TYPECLASS)
```

Comment:
Added this to load the BASE_SCRIPT_TYPECLASS class so we can `assert` (during `load_data()`) that the specified script class *is* a subclass.